pom.xml

```diff
@@ -285,4 +298,188 @@
       <url>https://oss.sonatype.org/service/local/staging/deploy/maven2/</url>
     </repository>
   </distributionManagement>
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>io.stargate.auth.api</groupId>
+        <artifactId>auth-api</artifactId>
+        <version>${project.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>io.stargate.auth</groupId>
+        <artifactId>authentication</artifactId>
+        <version>${project.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>io.stargate.auth.table</groupId>
+        <artifactId>auth-table-based-service</artifactId>
+        <version>${project.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>io.stargate.auth.table</groupId>
+        <artifactId>authentication</artifactId>
+        <version>${project.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>io.stargate.core</groupId>
+        <artifactId>core</artifactId>
+        <version>${project.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>io.stargate.cql</groupId>
+        <artifactId>cql</artifactId>
+        <version>${project.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>io.stargate.graphql</groupId>
+        <artifactId>graphqlapi</artifactId>
+        <version>${project.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>io.stargate.health</groupId>
+        <artifactId>health-checker</artifactId>
+        <version>${project.version}</version>
+      </dependency>
```

Comment:
Why are we also including top level services up here?

Reply:
Yeah I was debating that, "leaf" modules that no-one depend on don't need to be there. Maybe I was too lazy to find out which these are 🙂 

Possibly a moot point if we switch to `${project.version}` per your other suggestion.